src/components/UppyDashboard.tsx

```diff
@@ -1,36 +1,29 @@
-import React, { FC } from 'react';
+import React from 'react';
 import '@uppy/core/dist/style.css';
 import '@uppy/dashboard/dist/style.css';
 import { Dashboard } from '@uppy/react';
 import styled from 'styled-components';
-
-import { Uppy } from '../types/file.types';
+import { DashboardProps } from '@uppy/react/src/Dashboard';
 
 const StyledDashboard = styled.div`
   overflow: auto;
 `;
 
-interface UppyDashboardProps {
-  uppy: Uppy;
-}
-
 const uploaderMaxWidthPx = 10000;
 const uploaderMaxHeightPx = 200;
 
-const UppyDashboard: FC<UppyDashboardProps> = ({ uppy }) => {
-  return (
-    <StyledDashboard>
-      <Dashboard
-        uppy={uppy}
-        proudlyDisplayPoweredByUppy={false}
-        showSelectedFiles={false}
-        showProgressDetails
-        hideProgressAfterFinish
-        width={uploaderMaxWidthPx}
-        height={uploaderMaxHeightPx}
-      />
-    </StyledDashboard>
-  );
-};
+const UppyDashboard = (props: DashboardProps) => (
+  <StyledDashboard>
+    <Dashboard
+      proudlyDisplayPoweredByUppy={false}
```

Comment:
btw: er dette noe vi burde gjøre kanskje? 😜  kan vi komme i trøbbel hvis vi bruker deres produkt uten å gi cred?

Reply:
helt optional: https://uppy.io/docs/dashboard/#proudlyDisplayPoweredByUppy-true
men vi kan godt gjøre det for min del altså